redhat-director-scripts/docker/build_tripleo_containers.sh

```diff
@@ -26,41 +26,47 @@ declare -a openstack_releases=("train")
 
 declare -a openstack_platforms=("centos7" "centos8")
 
+declare -a base_containers=("docker.io/tripleotrain/centos-binary-nova-compute:current-tripleo-rdo" "docker.io/tripleotrain/centos-binary-nova-api:current-tripleo-rdo" "docker.io/tripleotrain/centos-binary-horizon:current-tripleo-rdo" "docker.io/tripleotraincentos8/centos-binary-nova-compute:current-tripleo-rdo" "docker.io/tripleotraincentos8/centos-binary-nova-api:current-tripleo-rdo" "docker.io/tripleotraincentos8/centos-binary-horizon:current-tripleo-rdo")
+
+for base_container in "${base_containers[@]}"
+do
+        docker pull ${base_container}
```

Comment:
use podman pull command. In train then use podman.

Reply:
changed to podman